.vsts-ci.yml

```diff
@@ -35,9 +35,9 @@ phases:
     agentOs: Linux
     dockerImage: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
     queue: 
-      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+      ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
         name: DotNetCore-Linux
-      ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:        
+      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI') }}:        
```

Comment:
change to "ne" and "PullRequest"?

Reply:
I'd just use "PullRequest" as the context switch for everything

Reply:
Sounds good. So notIn or neq PullRequest should be sufficient?

Reply:
Yes, it it's just a single value, I'd use "ne" PullRequest